### PR TITLE
chore: automate codegen

### LINKED ISSUE #571: regenerating all relevant pb files locally should be easy
Without having to manually install a list of go and python dependencies ...

### PR DESCRIPTION
**Pull request by wfxr**
Close #571

### CODE REVIEW

File: oomd/codegen/oomd.pb.go [hunk 1]
```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.27.1
-// 	protoc        v3.17.3
+// 	protoc-gen-go v1.26.0
```
Comment (YikSanChan) [inline oomd/codegen/oomd.pb.go:3]: any chance we have more up-to-date versions?

**Review by YikSanChan — APPROVED** [review]
lgtm

**Review by jinghancc — APPROVED** [review]
LGTM

**Review by lianxmfor — APPROVED** [review]
lgtm

File: oomd/codegen/oomd.pb.go (hunk 1 above)
Comment (wfxr) [reply oomd/codegen/oomd.pb.go:3]: Unless we modify and maintain this image

Comment (YikSanChan) [reply oomd/codegen/oomd.pb.go:3]: nvm, leave them as is

**Pull request merged**